tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -423,6 +423,7 @@
   "DD_TRACE_COMPUTE_STATS": "dd_trace_compute_stats",
   "DD_EXCEPTION_DEBUGGING_ENABLED": "dd_exception_debugging_enabled",
   "DD_EXCEPTION_DEBUGGING_MAX_FRAMES_TO_CAPTURE": "dd_exception_debugging_max_frames_to_capture",
+  "DD_EXCEPTION_REPLAY_CAPTURE_MAX_FRAMES": "dd_exception_replay_capture_max_frames",
```

Comment:
👍🏼

Reply:
If this is fundamentally the same feature as `DD_EXCEPTION_DEBUGGING_MAX_FRAMES_TO_CAPTURE` then it should probably map to the same config value. i.e. they're different values for the same configuration?